src/utils/setup/valence.rs

```diff
@@ -392,14 +457,16 @@ impl TestContext {
     fn tx_create_auctions_manager<'a>(
         &mut self,
         sender_key: &str,
+        chain: &str,
         min_auction_amount: impl AsRef<[(&'a str, MinAmount)]>,
         server_addr: impl AsRef<str>,
     ) -> Result<(), Error> {
         let mut contract_a: CosmWasm = self
             .get_contract()
             .contract(AUCTIONS_MANAGER_CONTRACT_NAME)
+            .src(chain)
             .get_cw();
-        let neutron = self.get_chain(NEUTRON_CHAIN_NAME);
+        let neutron = self.get_chain(chain);
```

Comment:
if it's chain agnostic let's change the name of the variable here.

Reply:
Ok! Fixed.